src/qt/bitcoin.cpp

```diff
@@ -252,7 +252,7 @@ public Q_SLOTS:
     QTimer *pollShutdownTimer;
 #ifdef ENABLE_WALLET
     PaymentServer* paymentServer;
-    WalletModel *walletModel;
+    std::vector<WalletModel*> walletModels;
```

Comment:
In commit "Qt: Load all wallets into WalletModels"

Might be simpler to make this a vector of unique_ptr, to avoid need for manual deletions. Also since renaming this member anyway, could follow current naming convenetion (m_wallet_models).

Reply:
Not sure unique_ptr is the right tool for the job here. Renamed.